global_oce_cs32/input_ad.sens/data.ecco

```diff
@@ -20,6 +20,13 @@
  gencost_name(2) = 'north10_flux_vol',
  gencost_outputlevel(2)=1,
  mult_gencost(2) = 1.,
+#
+ gencost_avgperiod(1)  = 'step',
```

Comment:
Shouldn't this be gencost_avgperiod(3), etc.?

Reply:
@timothyas
 One more thing, I need to change `wt_ones.data` to `wt_ones.bin` in data.ctrl to make it work and then the results is:
```
Y Y Y Y  0> 1< 1 FAIL  global_oce_cs32.sens
```
Is that expected? Probably, because the cost function changes. Maybe @jm-c can provide a reference results from the reference machine. Do you want me to push the `wt_ones.bin` fix?

Reply:
Hey @mjlosch thanks for these catches! I swear sometimes my code works ...😅
I'll go ahead and take care of the wt_ones.bin fix, thanks though

Reply:
As for the fail, I'm not sure. Might have something to do with the MOC code not being merged, but I'll test it right now.

Reply:
Is the test running a forward or adjoint simulation? I just ran this and it failed for the forward run because the grdchk package is turned on, which looks for adxx_* and can't find them. The adjoint run produces NaNs, but runs just fine.

I ran the adjoint both on master and on top of the MITgcm/MITgcm#405 branch, and it worked for both. Except when it ran on the master branch, the run just didn't recognize the MOC objective function.

Reply:
I didn't get any NaN.